mvc-web/README.md

```diff
@@ -0,0 +1,13 @@
+# MVC application for Ktor
+
+An application for adding and removing wishes to wishlist that uses Freemarker templates and Exposed.
+
+## Running
+
+Execute this command to run the sample:
+
+```bash
+./gradlew run
+```
+
+Then, navigate to [http://localhost:8080/](http://localhost:8080/) to see the sample home page.localhost:8080/](http://localhost:8080/) in a browser to create, edit, and delete articles.
```

Comment:
There's nothing at http://localhost:8080/. This line should say:

"Then, navigate to the following pages:
- [http://localhost:8080/wish/list](http://localhost:8080/wish/list) to view and modify the wishlist. 
- [http://localhost:8080/wish/topwishes](http://localhost:8080/wish/topwishes) to see the top three wishes in the list."